### ISSUE TITLE
ModelList equivalent for level names

### ISSUE THREAD

**Issue by Phanabani** [RatchetModding/replanetizer#45]
A way to translate `level##` to English level names so the user doesn't have to meticulously go through each level to find the one they want.

Possible options:
- Custom level select dialog
  - Select the root RaC data folder (and store it so the user can load the different levels at will)
  - Add a level selection frame with the English level names
    - Could be fun if we could find the planet icons to display in this list next to the names
  - Keep the manual `engine.ps3` file dialog in case users still need that
- Just add a simple popup that lists level name mappings

I think the first option would be nice to implement!

**Comment by MilchRatchet** [comment]
Good idea. Alongside that there is level independent data like gadgets or armors, those should probably then be removed from the Level class as they remain when switching levels between the same game. With that #43 would probably have to be done first as the solution to that one probably is to classify data by their origin. This origin would then have to be displayed to the user so that he understands what belongs to the individual levels and what is global to the game.
